# Publish all joint states -----------------------------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

# Joint velocity controllers ---------------------------------
motor_b_controller:
  type: velocity_controllers/JointVelocityController
  joint: wheel_back_joint
  pid: {p: 10000000.0, i: 1.0, d: 0.1}

motor_l_controller:
  type: velocity_controllers/JointVelocityController
  joint: wheel_left_joint
  pid: {p: 10000000.0, i: 1.0, d: 0.1}

motor_r_controller:
  type: velocity_controllers/JointVelocityController
  joint: wheel_right_joint
  pid: {p: 10000000.0, i: 1.0, d: 0.1}


gazebo_ros_control:
  pid_gains:
    wheel_back_joint:
      p: 10000000.0
      i: 1.0
      d: 0.1
    wheel_left_joint:
      p: 10000000.0
      i: 1.0
      d: 0.1
    wheel_right_joint:
      p: 10000000.0
      i: 1.0
      d: 0.1
